robots/cmd/labels-checker/main_test.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"github.com/google/go-github/github"
+	"testing"
+)
+
+func Test_checkAnyLabelExists(t *testing.T) {
+	type args struct {
+		prToCheck     *github.PullRequest
+		labelsToCheck []string
+	}
+	tests := []struct {
+		name string
+		args args
+		want bool
+	}{
+		{
+			name: "does not have any label to check",
+			args: args{
+				prToCheck:     &github.PullRequest{
+					Labels:              []*github.Label{
+						label("whatever"),
+						label("else"),
+						label("kind/enhancement"),
+						label("release-note-none"),
+					},
+				},
+				labelsToCheck: []string{
+					"lgtm",
+					"approved",
+					"kind/bug",
+				},
+			},
+			want: false,
+		},
+		{
+			name: "has two of three labels to check",
+			args: args{
+				prToCheck:     &github.PullRequest{
+					Labels:              []*github.Label{
+						label("lgtm"),
+						label("approved"),
+						label("needs-rebase"),
+						label("release-note-none"),
+					},
+				},
+				labelsToCheck: []string{
+					"lgtm",
+					"approved",
+					"kind/bug",
+				},
+			},
+			want: true,
+		},
+	}
```

Comment:
As you are checking "any" label from the list, one test is missing: The existence of one label out of 2 (or more).